interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.ts

```diff
@@ -37,5 +40,6 @@ export class DisasterTypeComponent implements OnInit, OnDestroy {
 
   public switchDisasterType(disasterType: DisasterType): void {
     this.disasterTypeService.setDisasterType(disasterType);
+    this.selectedDisasterType = disasterType.label;
```

Comment:
[Please use Disaster Type Key instead of Disaster Type Label here.](https://github.com/rodekruis/IBF-system/blob/master/interfaces/IBF-dashboard/src/app/models/country.model.ts#L44)

Reply:
Taken care